releng/org.eclipse.nebula.nebula-parent/pom.xml

```diff
@@ -364,6 +364,7 @@ Contributors:
 		<module>../../widgets/pgroup</module>
 		<module>../../widgets/progresscircle</module>
 		<module>../../widgets/pshelf</module>
+		<module>../../widgets/radiogroup</module>
```

Comment:
Don't the examples need to be re-parented?

Reply:
this change is non-functional, radiogroup is moved from `<!-- Mature -->` to `<!-- Incubation -->` section